applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -173,6 +249,41 @@ static uint32_t get_and_incr_seq_num(const struct bt_audio_stream *stream)
 	return 0;
 }
 
+static int update_sink_stream_qos(struct le_audio_headset *headset, uint32_t pres_delay_us)
+{
+	int ret;
+
+	if (headset->sink_stream.qos->pd != pres_delay_us) {
+		headset->sink_stream.qos->pd = pres_delay_us;
```

Comment:
This can be moved to line 274, right above 'ret = bt_audio_stream_qos(...'

Reply:
Not sure it can as we would be stopping and updating a receiver that already has this delay?

Reply:
I'm just talking about 'headset->sink_stream.qos->pd = pres_delay_us;', the if-statement would still guard against setting a pres delay to a receiver that already uses it. The reason for the move is that it's not needed if the playing_state is set.